lib/sycamore/sycamore/utils/aryn_config.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import yaml
+import logging
+
+_DEFAULT_PATH = os.path.join(pathlib.Path.home(), ".aryn", "config.yaml")
+
+
+class ArynConfig:
```

Comment:
I think what we really want here is a a singleton class.  It should probably read the config file once and keep the result.  I'd look into using class-variables for this.

Reply:
Changed.